packages/context/src/context.ts

```diff
@@ -349,6 +355,120 @@ export class Context extends EventEmitter {
     return this;
   }
 
+  /**
+   * Create a corresponding binding for configuration of the target bound by
+   * the given key in the context.
+   *
+   * For example, `ctx.configure('controllers.MyController').to({x: 1})` will
+   * create binding `controllers.MyController:$config` with value `{x: 1}`.
+   *
+   * @param key The key for the binding to be configured
+   */
+  configure<ConfigValueType = BoundValue>(
+    key: BindingAddress = '',
+  ): Binding<ConfigValueType> {
+    const keyForConfig = BindingKey.buildKeyForConfig(key);
+    const bindingForConfig = this.bind<ConfigValueType>(keyForConfig).tag({
```

Comment:
See my comments above, I think we should pass `ConfigValueType` to the function creating the binding key. Thoughts?

```ts
const keyForConfig = BindingKey.buildKeyForConfig< ConfigValueType>(key);
const bindingForConfig = this.bind(keyForConfig)./*etc*/
```

Reply:
> See my comments above, I think we should pass ConfigValueType to the function creating the binding key. Thoughts?

☝️